### PR TITLE
fix: changing Settings.jsx frontend text color to white

### LINKED ISSUE #401: [OTHER] Color of the text on the setting page should be more bright
### What would you like to share?

Below I underlined the text with yellow line

Capture

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by AaadityaG**
# Fixes Issue

**My PR closes #401**

# 👨‍💻 Changes proposed(What did you do ?)
Adding inline CSS to the three Form.HelpText component.

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

# 📷 Screenshots

###  Before : 
Whisper _ Chat Application - Google Chrome 15-10-2023 04_51_49 PM

###  After : 
Whisper _ Chat Application — Mozilla Firefox 15-10-2023 04_14_45 PM

### CODE REVIEW

File: client/src/pages/Settings.jsx
Comment (Dun-sin) [inline client/src/pages/Settings.jsx:1]: use tailwind

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/pages/Settings.jsx
Comment (AaadityaG) [reply client/src/pages/Settings.jsx:1]: Done Sir!

File: client/src/pages/Settings.jsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const Searching = () => {
                                 />
                                 <Form.ControlLabel className='text-white text-base font-normal'>MODE</Form.ControlLabel>
                             </div>
-                            <Form.HelpText className='text-zinc-600 text-xs'>
+                            <Form.HelpText className='text-zinc-600 text-xs decoration-white'>
```
Comment (Dun-sin) [inline client/src/pages/Settings.jsx:81]: Wrong tailwind class

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/pages/Settings.jsx (hunk 1 above)
Comment (AaadityaG) [reply client/src/pages/Settings.jsx]: My bad...😑

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Pull request merged**